buildpacks/java-maven/bin/build

```diff
@@ -51,20 +52,20 @@ version = "${jdk_version}"
 url = "${jdk_url}"
 EOF
 
-  echo "$layers_dir"/jdk > "$layers_dir"/jdk/env/JAVA_HOME
+  echo "$jdk_layer_dir" > "$jdk_layer_dir"/env/JAVA_HOME
   if [[ -z ${LD_LIBRARY_PATH} ]]; then
     echo "${JAVA_HOME}/jre/lib/amd64/server" > ${jdk_layer_dir}/env/LD_LIBRARY_PATH
   else
     echo "${JAVA_HOME}/jre/lib/amd64/server:${LD_LIBRARY_PATH}" > ${jdk_layer_dir}/env/LD_LIBRARY_PATH
   fi
 
-  mkdir -p ${jdk_layer_dir}/profile.d
-  cat > "${jdk_layer_dir}/profile.d/jdk.sh" << EOF
-export JAVA_HOME=${jdk_layer_dir}
+  mkdir -p ${jdk_layer_dir}/exec.d
+  cat > "${jdk_layer_dir}/exec.d/jdk.sh" << EOF
+echo JAVA_HOME=${jdk_layer_dir}
```

Comment:
Why has this been changed to echo?

Reply:
Because the spec states that env vars should be output as key-value pairs -- https://github.com/buildpacks/spec/blob/main/buildpack.md#execd and https://github.com/buildpacks/spec/blob/main/buildpack.md#execd-output-toml.  What was missing is a redirection to "FD 3".

Reply:
I still don't see the redirection to fd3?

Reply:
It's in https://github.com/buildpacks/samples/pull/151/commits/5b94deb0ecb1cced307770d0152beb747bb9c347 . I think this is GH being slow to update.